README.md

```diff
@@ -17,6 +17,10 @@ scrapy crawl -L WARNING otomoto -o otomoto.json
 This will generate `otomoto.json` file with all cars that are currently
 available.  You can further investigate them or create some analysis.
 
+You can change scope of search by changing url in otomoto.py file
+```
+start_urls = ['https://www.otomoto.pl/osobowe/']
+```
```

Comment:
Maybe instead of adding stuff to documentation, it would be better to allow specifying that from cmd line parameters.

https://docs.scrapy.org/en/latest/topics/spiders.html#spider-arguments